postgres/init.sls

```diff
@@ -8,6 +8,11 @@ include:
 {{ postgres.conf_dir }}:
   file.directory:
     - makedirs: True
+    - user: postgres
+    - group: postgres
+    - dir_mode: 700
+    - clean: True
```

Comment:
This is probably a (backwards incompatible) change that should be discussed with the user base.